### PR TITLE
Fix export issues with encounter, provider, medication

### PR DESCRIPTION
**Pull request by tstrass**
This fixes template and model issues that were discovered during
export integration with cypress.

### CODE REVIEW

File: models/value_set.go [hunk 1]
```diff
@@ -83,7 +83,10 @@ func (v ValueSetMap) OidForCode(codedValue CodedConcept, valuesetOids []string)
 func codeSetContainsCode(sets []CodeSet, codedValue CodedConcept) bool {
 	for _, cs := range sets {
 		for _, val := range cs.Values {
-			if val.CodeSystem == codedValue.CodeSystem && val.Code == codedValue.Code {
+			if (val.CodeSystem == codedValue.CodeSystem ||
+				val.CodeSystemName == codedValue.CodeSystemName ||
+				val.CodeSystemName == codedValue.CodeSystem) &&
```
Comment (ljades) [inline models/value_set.go:88]: Is this supposed to be `val.CodeSystem == codedValue.CodeSystem`?

Comment (tstrass) [reply models/value_set.go:88]: No. This condition has to be here due to an idiosyncrasy of the bundle

**Review by ljades — APPROVED** [review]
Looks to me, in that case.

**Review by okeefm — APPROVED** [review]
Looks good to me.

**Pull request merged**